tests/BreadcrumbsTest.php

```diff
@@ -232,6 +232,136 @@ public function testAddCrumb($crumbs)
+    public function testIsAbleToEnchainAddCrumbMethod($crumbs)
+    {
+        $b = new Breadcrumbs;
+        $n = count($crumbs);
+
+        switch ($n) {
+            case 1:
+                $b->addCrumb($crumbs[0]);
+
+                break;
+
+            case 2:
+                $b->addCrumb($crumbs[0])
+                    ->addCrumb($crumbs[1]);
+
+                break;
+
+            case 3:
+                $b->addCrumb($crumbs[0])
+                    ->addCrumb($crumbs[1])
+                    ->addCrumb($crumbs[2]);
+
+                break;
+
+            case 4:
+                $b->addCrumb($crumbs[0])
+                    ->addCrumb($crumbs[1])
+                    ->addCrumb($crumbs[2])
+                    ->addCrumb($crumbs[3]);
+
+                break;
+
+            case 5:
+                $b->addCrumb($crumbs[0])
+                    ->addCrumb($crumbs[1])
+                    ->addCrumb($crumbs[2])
+                    ->addCrumb($crumbs[3])
+                    ->addCrumb($crumbs[4]);
+
+                break;
+
+            default:
+                // 0? or > 5?  TODO: add cases
+                break;
+        }
+
+        $this->assertCount($n, $b->getBreadcrumbs());
+
+        // TODO: Add output test?
+    }
+
+
+    /**
+     * @testdox `add` can be used as an alias for `addCrumb` method
+     * @depends testAddCrumb
+     * @dataProvider crumbsProvider
+     */
+    public function testAddCrumbAlias($crumbs)
+    {
+        $b = new Breadcrumbs;
+
+        foreach ($crumbs as $crumb) {
+            $b->add($crumb);
+        }
+
+        $this->assertCount(count($crumbs), $b->getBreadcrumbs());
+    }
+
+    /**
+     * @testdox Is able to enchain `add` method
+     * @depends testIsAbleToEnchainAddCrumbMethod
+     * @dataProvider crumbsProvider
+     */
+    public function testIsAbleToEnchainAddMethod($crumbs)
+    {
+        $b = new Breadcrumbs;
+        $n = count($crumbs);
+
+        switch ($n) {
+            case 1:
+                $b->add($crumbs[0]);
+
+                break;
+
+            case 2:
+                $b->add($crumbs[0])
+                    ->add($crumbs[1]);
+
+                break;
+
+            case 3:
+                $b->add($crumbs[0])
+                    ->add($crumbs[1])
+                    ->add($crumbs[2]);
+
+                break;
+
+            case 4:
+                $b->add($crumbs[0])
+                    ->add($crumbs[1])
+                    ->add($crumbs[2])
+                    ->add($crumbs[3]);
+
+                break;
+
+            case 5:
+                $b->add($crumbs[0])
+                    ->add($crumbs[1])
+                    ->add($crumbs[2])
+                    ->add($crumbs[3])
+                    ->add($crumbs[4]);
+
+                break;
+
+            default:
+                // 0? or > 5?  TODO: add cases
+                break;
+        }
+
+        $this->assertCount($n, $b->getBreadcrumbs());
+
+        // TODO: Add output test?
```

Comment:
No need, we have other methods that test output, so if something is wrong there, those tests will fail.